simulators/history/portal-interop/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 
 [dependencies]
 ethportal-api = { git = "https://github.com/ethereum/trin", rev = "2a32224e3c2b0b80bc37c1b692c33016371f197a" }
+portal-spec-test-utils-rs = { git = "https://github.com/ethereum/portal-spec-tests", rev = "d1e996d0d4dc2136b3cd38d9e25cdc3a6b74dcd9" }
```

Comment:
I'm curious, but is storing these constants in a separate repository necessary? Seems to me like we'd like to store them in this repo, to make it easier to add/remove/edit constants, rather than juggling them between multiple repositories. Is there any reason why we couldn't just create a "utils" workspace inside `portal-hive`?

Reply:
Maybe we can store it in the simulator folder I will check

Reply:
Honestly, I'm using these constants right now in some scripting work I'm doing in trin, so maybe we just store them in `trin-utils`?